metadata-ingestion/src/datahub/ingestion/source/sql/mssql/source.py

```diff
@@ -179,6 +195,14 @@ def __init__(self, config: SQLServerConfig, ctx: PipelineContext):
         self.table_descriptions: Dict[str, str] = {}
         self.column_descriptions: Dict[str, str] = {}
         self.stored_procedures: FileBackedList[StoredProcedure] = FileBackedList()
+
+        self.report = SQLSourceReport()
```

Comment:
iirc this line shouldn't be necessary?

Reply:
This is probably something to be revisited at some moment because parent class `SqlAlchemySource` sets report as: `self.report: SQLSourceReport = SQLSourceReport()`. Also, I added this line because unit tests were complaining about non-initialized report instance (as I was mocking the parent).